gateway-service/src/test/java/org/zowe/apiml/gateway/security/service/AuthenticationServiceTest.java

```diff
@@ -208,7 +208,7 @@ public void shouldParseJwtTokenAsQueryResponse() {
         assertEquals(DOMAIN, parsedToken.getDomain());
         assertEquals(USER, parsedToken.getUserId());
         assertEquals(parsedToken.getCreation().toString().substring(0, 16), dateNow);
-        Date toBeExpired = DateUtils.addDays(parsedToken.getCreation(), 1);
+        Date toBeExpired = DateUtils.addHours(parsedToken.getCreation(), 24);
```

Comment:
This was necessary to make it work on March 28 before the daylight savings time changed.